docs/Localiser/index.md

```diff
@@ -0,0 +1,47 @@
+# Localiser
+A localising system for networked messages. 
+
+## Creating and Using
+
+The following example is using strings found in the DayZ stringtable. 
+
+The localiser class has overloaded the array assignment operator to allow setting of the string table keys and placing in other variables at the desired format.
+
+```csharp
+CF_Localiser localiser = new CF_Localiser("STR_DATE_FORMAT");
```

Comment:
Put a comment on this at the end with the value of `STR_DATE_FORMAT` so you can see where the placeholders are and understand the example better.